executor/hnswlib_searcher.py

```diff
@@ -32,13 +32,14 @@ def __init__(
         metric: str = DEFAULT_METRIC,
         dim: int = 0,
         max_elements: int = 1_000_000,
-        ef_construction: int = 400,
+        ef_construction: int = 200,
         ef_query: int = 50,
-        max_connection: int = 64,
+        max_connection: int = 16,
         dump_path: Optional[str] = None,
         traversal_paths: Iterable[str] = ('r',),
         is_distance: bool = True,
         last_timestamp: datetime = datetime.min,
+        num_threads: int = -1,
```

Comment:
add docstr for `num_threads`

Reply:
It doesn't matter here really. It's documented in the main class